<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewCategoryDetailsDialog</class>
 <widget class="QDialog" name="ViewCategoryDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 192, 196);</string>
  </property>
  <widget class="QLabel" name="categoryTitle_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft PhagsPa</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Category Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="categoryType_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Hebrew</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Category Type:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="categoryDescription_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro Light</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="plainText">
    <string>CATEGORY DESCRIPTION</string>
   </property>
  </widget>
  <widget class="QLabel" name="color_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="categoryTypeText_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Hebrew</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
